core/src/main/java/tc/oc/pgm/destroyable/DestroyableFactory.java

```diff
@@ -10,15 +11,37 @@
 import tc.oc.pgm.modes.Mode;
 import tc.oc.pgm.teams.TeamFactory;
 import tc.oc.pgm.util.material.MaterialMatcher;
+import tc.oc.pgm.util.xml.InvalidXMLException;
 
 @FeatureInfo(name = "destroyable")
 public class DestroyableFactory extends ProximityGoalDefinition {
+  public static enum SparksType {
```

Comment:
To keep the behavior of `true` and `false` you can implement the `Aliased` interface in your enum to give them aliases:
```
  @NotNull
  @Override
  public Iterator<String> iterator() {
    return Iterators.forArray(name(), name);
  }
```